godot-core/src/builtin/array.rs

```diff
@@ -675,6 +675,26 @@ impl<T: GodotType> fmt::Debug for Array<T> {
     }
 }
 
+impl<T: GodotType + fmt::Display> fmt::Display for Array<T> {
+    /// Formats `Array` to match Godot's string representation
```

Comment:
```suggestion
    /// Formats `Array` to match Godot's string representation.
```

(We should possibly change this elsewhere too, but we adopted this convention of full sentences only after some time).

Reply:
Updating all instances in my comments to have the full sentences paradigm.